contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1251,6 +1262,48 @@ SendColumnMetadataToken(int natts, bool sendRowStat)
 			}
 		}
 
+		/* For Spatial Types */
+		/*
+			* Check if it is spatial Data type
+			* Send the Corresponding MetaData Columns
+		*/			
+		if (col->isSpatialType)
+		{
+			
+			/* Current Database Name and Length are expected by the Driver */
+			db_name = pltsql_plugin_handler_ptr->get_cur_db_name();
+			temp8 = (uint8_t) pg_mbstrlen(db_name);
+			resetStringInfo(&tempBuf);
+			TdsUTF8toUTF16StringInfo(&tempBuf, db_name,
+									strlen(db_name));
+			TdsPutbytes(&temp8, sizeof(temp8));
+			TdsPutbytes(tempBuf.data, tempBuf.len);
+
+			/* Since Schema Name is always sys in Babelfish Server we can directly send it */
+			temp8 = (uint8_t) pg_mbstrlen("sys");
+			resetStringInfo(&tempBuf);
+			TdsUTF8toUTF16StringInfo(&tempBuf, "sys",
+									strlen("sys"));
```

Comment:
Did you check how domains are sent on the wire?

Reply:
We checked the domains, but TSQL doesn't support these for spatial types and currently **babelfish do support** this, but this is not impacting the feature it's just that we need to make it similar to TSQL, which will be done in upcoming PR's.
I have also added a test for this in TestSpatialPoint-vu-*